src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -456,7 +456,46 @@ void submitStreamOperation(final PathOperation streamOp, final OpType opType) {
 	}
 
 	void submitStreamCreateOperation(final PathOperation streamOp, final String nodeAddr) {
-		// TODO: Vlad, issue SDP-47
+		try {
+			val endpointUri = endpointCache.computeIfAbsent(nodeAddr, this::createEndpointUri);
+			val controller = controllerCache.computeIfAbsent(endpointUri, this::createController);
+			val scopeName = DEFAULT_SCOPE;
+			var streamName = streamOp.item().name();
+			if(streamName.startsWith(SLASH)) {
```

Comment:
Let's move this duplication code (stream name fixing) into the some method

Reply:
Moved